metaci/build/tests/test_flows.py

```diff
@@ -72,7 +74,11 @@ def test_post_task__result_has_exception(get_spec):
 
 
 @pytest.mark.django_db
-def test_post_task__single_robot_task(get_spec):
+def test_post_task__single_robot_task(mocker, get_spec):
+    mocker.patch(
+        "metaci.build.flows.settings",
+        RESULT_EXPORT_ENABLED=False,
```

Comment:
Why do you need this, given that the default is False?

Reply:
So that I can test it locally having it set to True and still have my local tests pass, do you have an alternative @davisagli ?

Reply:
We should probably figure out whether we intend for the test suite to be run with or without your local "env" file loaded. If not, this isn't needed. If so, it would probably be better to have a session-scoped pytest fixture that resets the environment so that it doesn't leak into tests. We don't need to tackle that in this PR though.